fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/NbtIdentifier.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019, 2020 FabricMC
```

Comment:
Half has done the forbidden and added 2020 to this license header, absolutely unacceptable!

Reply:
Change to 2019

Reply:
Actually, don't do that, write a script to change copyright date in all fabric projects, only then this PR is valid!

Reply:
ohno I included the year in my pr we're all going to die corb